content/metadata.yaml

```diff
@@ -565,3 +565,34 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-06-23
+-
+    github: kevinsunofficial
+    name: YuCheng Sun
+    initials: YS
+    orcid: 
+    email: ys4aj@virginia.edu
+    contributions:
+      - Writing - Figure Drafting
+    affiliations:
+      - Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
+    code of conduct:
+      confirmed: yes
+    coi:
+      string: "None"
+      lastapproved: !!str 2020-?-?
```

Comment:
```suggestion
      lastapproved: !!str 2020-07-09
```

Reply:
Yes. I made these figures and will keep improving them.